src/main/java/net/sf/jabref/external/FindFullTextAction.java

```diff
@@ -59,6 +58,7 @@ public void init() throws Throwable {
     public void run() {
         if (basePanel.getSelectedEntries().size() != 1) {
             basePanel.output(Localization.lang("This operation requires exactly one item to be selected."));
+            result = Optional.empty();
```

Comment:
I think it needs to be reset. If not, if you run it once and then try to run it with multiple entries selected, update will still be called and then result is not empty.